scripts/src/test.sh

```diff
@@ -70,13 +70,8 @@ gtest() {
 }
 
 # $1 - ROOT DIR
-rtest_debug() {
-  cargo run --package gear-node $CARGO_FLAGS -- runtime-spec-tests "$1"/gear-test/spec/*.yaml -l0
```

Comment:
Why `cargo` call has been removed?

Reply:
because `cargo run` rebuilds wasm runtime each time which adds ~2min.

Reply:
It's ok for CI but not for user who just runs `./scripts/gear.sh test rtest`

Reply:
I think user must do builds first then. demo from examples are not builds in cargo run also as you can see

Reply:
> cargo run rebuilds wasm runtime each time

Why this even happens?

Reply:
It means `substrate` overuse `cargo:rerun-if-changed` so it might be better to patch `substrate`